src/pages/Transfer/CrossChainTransferForm/index.tsx

```diff
@@ -0,0 +1,59 @@
+
+import { useTranslation } from 'react-i18next';
+import { withErrorBoundary } from 'react-error-boundary';
+import { useSelector } from 'react-redux';
+
+import { COLLATERAL_TOKEN_SYMBOL } from 'config/relay-chains';
+import { StoreType } from 'common/types/util.types';
+import { displayMonetaryAmount } from 'common/utils/utils';
+import AvailableBalanceUI from 'components/AvailableBalanceUI';
+import Accounts from 'components/Accounts';
+import Chains from 'components/Chains';
+import TokenField from 'components/TokenField';
+import ErrorFallback from 'components/ErrorFallback';
+import FormTitle from 'components/FormTitle';
+import SubmitButton from 'components/SubmitButton';
+
+const CrossChainTransferForm = (): JSX.Element => {
+  const { t } = useTranslation();
+
+  const {
+    collateralTokenTransferableBalance
+  } = useSelector((state: StoreType) => state.general);
+
+  return (
+    <form className='space-y-8'>
+      <FormTitle>
+        {t('transfer_page.cross_chain_transfer_form.title')}
+      </FormTitle>
+      <div>
+        <AvailableBalanceUI
+          label={t('transfer_page.cross_chain_transfer_form.balance')}
+          balance={displayMonetaryAmount(collateralTokenTransferableBalance)} />
+        <div>
```

Comment:
@tomjeatt 
This `div` looks redundant.

Reply:
Not unfortunately :) removing the causes some layout issues. I'm likely to need to refactor the layout a little when I introduce the form logic, so if I have time I'll find a better way of handling the layout issues (same applies to other div comments but won't repeat).